Hudl.Mjolnir/Command/BulkheadInvoker.cs

```diff
@@ -64,7 +65,7 @@ public BulkheadInvoker(IBreakerInvoker breakerInvoker, IBulkheadFactory bulkhead
             var executedHere = false;
             try
             {
-                if (_config.GetConfig("mjolnir.useCircuitBreakers", true))
```

Comment:
Where are we re-wiring these config values in? These are all publicly-used values with the library. Do we need an implementation somewhere that continues to use them?

Reply:
See https://github.com/hudl/Mjolnir/wiki/Configuration-Keys